css/properties/border-style.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "properties": {
+      "border-style": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-style",
+          "support": {
+            "webview_android": {
+              "version_added": "2.6"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": "12"
```

Comment:
Same as above. The original table notes `20 or earlier (12)` for the Edge browsers; a closer review here would be appreciated.